extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigation_index.php

```diff
@@ -23,20 +23,23 @@
 						'wds-icons-magnifying-glass',
 						'wds-icon wds-icon-small'
 					) ?>
-					<input class="wds-global-navigation__search-input" name="search" placeholder="<?= DesignSystemHelper::renderText( $model['search']['module']['placeholder-inactive'] ); ?>"/>
+					<input class="wds-global-navigation__search-input"
+						name="<?= Sanitizer::encodeAttribute( $model['search']['module']['results']['param-name'] ); ?>"
+						data-active-text="<?= DesignSystemHelper::renderText( $model['search']['module']['placeholder-active'] ); ?>"
```

Comment:
`data-active-text` might be a little misleading. I suggest we put `placeholder` somewhere in the name, don't have any great ideas though.